docs/doc/11-integrations/30-access-tool/01-bendsql.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: bendsql
-sidebar_label: bendsql
+title: BendSQL
```

Comment:
What's the official name for bendsql? We used to use `bendsql` everywhere in the doc. If it is changed, we need change globally.

Reply:
It's `BendSQL`